service-registry/src/test/resources/config/frameworkVersion.properties

```diff
@@ -0,0 +1 @@
+FRAMEWORK_VERSION=0.6.0
```

Comment:
How about change the version to xxx-test?

Reply:
The new modification deletes this file,replace with pom.properties